packages/docs/getting-started/faucet.md

```diff
@@ -30,19 +30,25 @@ You can also use the Celo Blockchain client to create and manage account keypair
 
 Create and cd into the directory where you want to store the keypair. You can name this whatever you’d like, but here’s a default you can use:
 
-`$ mkdir celo-data-dir $ cd celo-data-dir`
+```bash
+mkdir celo-data-dir $ cd celo-data-dir
+```
 
 Create an account by running this command:
 
-`` $ docker run -v `pwd`:/root/.celo -it us.gcr.io/celo-testnet/celo-node:alfajores account new ``
+```bash
+docker run -v `pwd`:/root/.celo -it us.gcr.io/celo-testnet/celo-node:alfajores account new
+```
 
 It will prompt you for a passphrase, ask you to confirm it, and then will output your account address:
 
 `Address: <YOUR-ACCOUNT-ADDRESS>`
 
 This creates a keypair and stores it. Save this address to an environment variable, so that you can reference it later:
 
-`$ export CELO_ACCOUNT_ADDRESS=<YOUR-ACCOUNT-ADDRESS>`
+```bash
```

Comment:
Not in this PR, but I wonder if we should properly namespace this since this should only apply to Alfajores?